### PR TITLE
Validation: Fix missing descriptions 

### PR DESCRIPTION
**Pull request by annmariehitchcock**
This PR addresses the following issue:
description-missing
 
Fixes 30 issues across 30 files.

### CODE REVIEW

**Comment by annmariehitchcock** [comment]
Global effort to fix validation errors

@adegeo

The Content & Learning team is fixing build validation errors on docs.microsoft.com. This effort will eliminate potential accessibility, security, and usability issues. This PR includes only build validation fixes and does not change other content. I’d very much appreciate your timely review and feedback on these changes I’ve suggested. Thanks!

File: dotnet-desktop-guide/framework/wpf/controls/textblock.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "TextBlock"
+description: Learn how to use TextBlock control to provide flexible text support for UI scenarios that do not require more than one paragraph of text.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/textblock.md:3]:
```suggestion
description: Learn how to use the TextBlock control to provide flexible text support for UI scenarios that do not require more than one paragraph of text.
```

File: dotnet-desktop-guide/framework/wpf/controls/toolbar-overview.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "ToolBar Overview"
+description: Overview of how the ToolBar controls are containers for a group of commands or controls which are typically related in their function.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/toolbar-overview.md:3]:
```suggestion
description: Overview of how ToolBar controls are containers for a group of commands or controls which are typically related in their function.
```

File: dotnet-desktop-guide/framework/wpf/controls/tooltip-overview.md [hunk 3]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "ToolTip Overview"
+description: Overview introducing the tooltip and how to create and customize tooltip content in Windows Presentation Foundation (WPF) applications.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/tooltip-overview.md:3]:
```suggestion
description: Overview that introduces how to create and customize tooltip content in Windows Presentation Foundation (WPF) applications.
```

File: dotnet-desktop-guide/framework/wpf/controls/walkthroughs-create-a-custom-animated-button.md [hunk 4]
```diff
@@ -1,5 +1,6 @@
 ---
 title: "Walkthroughs: Create a Custom Animated Button"
+description: Walkthrough how to create a custom animated button using a style and template so you can apply this custom button easily to any buttons in your application.
```
Comment (adegeo) [inline dotnet-desktop-guide/framework/wpf/controls/walkthroughs-create-a-custom-animated-button.md:3]:
```suggestion
description: A walkthrough about how to create a custom animated button using a style and template so you can apply this custom button easily to any buttons in your application.
```

**Review by adegeo — APPROVED** [review]

**Pull request merged**